src/main/java/de/retest/recheck/ui/review/ActionChangeSet.java

```diff
@@ -14,16 +16,23 @@
 	private final String description;
 	private final String goldenMasterPath;
 	private final ScreenshotChanges screenshots;
+	private final Map<String, String> metadata;
 
 	public ActionChangeSet() {
-		this( null, null, ScreenshotChanges.empty() );
+		this( null, null, ScreenshotChanges.empty(), new HashMap<>() );
 	}
 
 	public ActionChangeSet( final String description, final String goldenMasterPath,
 			final ScreenshotChanges screenshots ) {
+		this( description, goldenMasterPath, screenshots, new HashMap<>() );
```

Comment:
I would always prefer `Collections.emptyMap()` for such cases to avoid creating multiple instances.